### ISSUE TITLE
New function often leads user back to landing page

### ISSUE THREAD

**Issue by PropGit** [parallaxinc/solo#275]
Reported by an instructor:

When first going to Solo (from the BP Solo button on BP Launcher [Win10]), clicking New, entering details, and clicking the Continue button leads user back to the landing page quite often.  3 out of 4 times (for example).

This _may_ be a cache issue.  @PropGit will confirm week of 2/17 when user can experiment again.

**Comment by zfi** [comment]
I am not clear on what is actually getting launched when the BP Solo button is clicked. Is this a Chromium browser session separate from the browsers that are installed on the client's machine?

**Comment by PropGit** [comment]
Yes, by nature of nwjs, the full Chromium engine is included with the nwjs-wrapped BlocklyProp Launcher for Windows and Mac - any URLs referenced by the app (or by user action in this case, clicking on a button that references a URL) opens up a Chromium-based browser window using the included engine.  In this case, the actual Google Chrome Browser need not be present on the user's Win/Mac system at all, and the included Chromium version is static (not auto-updated at Google's will; but rather updated only at our will upon future releases of Launcher for Win/Mac).

**Comment by PropGit** [comment]
Re: "This may be a cache issue. @PropGit will confirm week of 2/17 when user can experiment again."

The user (instructor) did not get back to me yet.
